{"uuid": "618fc712-82d2-4b7e-b9ad-a6ceee0b65f8", "counter": 118, "stdout": "\u001b[0;34m<192.168.38.8> (1, b'\\r\\n{\"changed\": true, \"stdout\": \"\", \"stderr\": \"python3: can\\'t open file \\'/home/vmadm/southbound/subnetting.py\\': [Errno 2] No such file or directory\", \"rc\": 2, \"cmd\": [\"python3\", \"southbound/subnetting.py\", \"192.168.100.0/24\"], \"start\": \"2024-04-21 21:11:35.816485\", \"end\": \"2024-04-21 21:11:35.869284\", \"delta\": \"0:00:00.052799\", \"failed\": true, \"msg\": \"non-zero return code\", \"invocation\": {\"module_args\": {\"_raw_params\": \"python3 southbound/subnetting.py 192.168.100.0/24\", \"_uses_shell\": false, \"stdin_add_newline\": true, \"strip_empty_ends\": true, \"argv\": null, \"chdir\": null, \"executable\": null, \"creates\": null, \"removes\": null, \"stdin\": null}}}\\r\\n', b'Shared connection to 192.168.38.8 closed.\\r\\n')\u001b[0m", "start_line": 108, "end_line": 109, "runner_ident": "172e0e39-e92b-46e0-8924-111790a94540", "event": "verbose", "pid": 3614295, "created": "2024-04-21T21:11:35.939811", "parent_uuid": "52540038-3809-6569-fdf5-000000000005", "event_data": {"playbook": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml", "playbook_uuid": "3ef4e3f8-cbae-4f32-a029-d01517f8bcb9", "play": "Deploy VPC Infrastructure with network namespace", "play_uuid": "52540038-3809-6569-fdf5-000000000002", "play_pattern": "hostvms", "task": "Get /30 subnet for current VPC", "task_uuid": "52540038-3809-6569-fdf5-000000000005", "task_action": "command", "resolved_action": "ansible.builtin.command", "task_args": "", "task_path": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml:11", "uuid": "618fc712-82d2-4b7e-b9ad-a6ceee0b65f8"}}